pkg/controller/controller.go

```diff
@@ -833,3 +838,28 @@ func (c *Controller) DeleteVolumeGroup(contx context.Context, opt *pb.DeleteVolu
 
 	return pb.GenericResponseResult(nil), nil
 }
+func (c *Controller) GetMetrics(context context2.Context, opt *pb.GetMetricsOpts) (*pb.GenericResponse, error) {
```

Comment:
I am wondering what's the difference between `context` with `golang.org/x/net/context`, shall we move all context to the former one?

Reply:
Sounds good to go. found a similar thing done in K8S
https://github.com/kubernetes/kubernetes/issues/60560